book-club-ui/src/views/BookView.vue

```diff
@@ -43,6 +46,22 @@ export default class BookView extends Vue {
     this.loading = false;
   }
 
+  async markBookAsReadForUser(){
+    this.loading = true;
+    const store = userStore();
+    const user = store.user;
+    user.booksRead.push(this.book);
+    const userService = new UserService();
+    const updatedUser = await userService.updateUser(user);
+    store.$patch({user: updatedUser});
+    this.loading = false;
+    notify({
+      title: `Successfully updated user`,
```

Comment:
Should probably be "Book marked as finished" or something similar

Reply:
Have pushed out an update to fix this. :)